PocketKit/Sources/Textile/Style/Style+Definitions.swift

```diff
@@ -53,23 +50,39 @@ public extension Style {
             public let p3 = Style(family: .graphik, size: .p3, weight: .regular)
             public let p4 = Style(family: .graphik, size: .p4, weight: .regular)
             public let p5 = Style(family: .graphik, size: .p5, weight: .regular)
-            public let p6 = Style(family: .graphik, size: .p6, weight: .regular) // used in widgets
-            public let w8 = Style(family: .graphik, size: .p4, weight: .medium) // used in widgets
+            // widgets
+            public let p6 = Style(family: .graphik, size: .p6, weight: .regular)
+            public let w8 = Style(family: .graphik, size: .p4, weight: .medium)
         }
 
-        public struct Serif {
-            public let title = Style(family: .blanco, size: .title, weight: .bold)
-            public let h1 = Style(family: .blanco, size: .h1, weight: .bold)
-            public let h2 = Style(family: .blanco, size: .h2, weight: .bold)
-            public let h3 = Style(family: .blanco, size: .h3, weight: .bold)
-            public let h4 = Style(family: .blanco, size: .h4, weight: .bold)
-            public let h5 = Style(family: .blanco, size: .h5, weight: .bold)
-            public let h6 = Style(family: .blanco, size: .h6, weight: .bold)
-            public let p1 = Style(family: .blanco, size: .p1, weight: .regular)
-            public let p2 = Style(family: .blanco, size: .p2, weight: .regular)
-            public let p3 = Style(family: .blanco, size: .p3, weight: .regular)
-            public let p4 = Style(family: .blanco, size: .p4, weight: .regular)
-            public let p5 = Style(family: .blanco, size: .p5, weight: .regular)
+        public struct TextStyles {
```

Comment:
Maybe update `TextStyles` to `GenericTextStyles` to match the font styling type?